samza-core/src/test/java/org/apache/samza/execution/TestExecutionUtils.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.execution;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.apache.samza.Partition;
+import org.apache.samza.application.StreamApplication;
+import org.apache.samza.config.Config;
+import org.apache.samza.job.ApplicationStatus;
+import org.apache.samza.operators.functions.JoinFunction;
+import org.apache.samza.runtime.AbstractApplicationRunner;
+import org.apache.samza.runtime.ApplicationRunner;
+import org.apache.samza.system.SystemAdmin;
+import org.apache.samza.system.SystemStreamMetadata;
+import org.apache.samza.system.SystemStreamPartition;
+
+public class TestExecutionUtils {
+
+  static ApplicationRunner createRunner(Config config) {
+    return new AbstractApplicationRunner(config) {
+      @Override
+      public void run(StreamApplication streamApp) {
+      }
+
+      @Override
+      public void kill(StreamApplication streamApp) {
+
+      }
+
+      @Override
+      public ApplicationStatus status(StreamApplication streamApp) {
+        return null;
+      }
+    };
+  }
+
+  static JoinFunction createJoin() {
```

Comment:
Also nothing to do with Operators, it's just some common code b/w ExecutionPlanner layer (and below) test classes. One of these methods (createRunner) will probably be irrelevant after the refactor in #92. Also, @xinyuiscool why not use mocks for these instead of creating actual instances?